android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -986,6 +986,23 @@ abstract class EngineSession(
      */
     abstract fun requestTranslationRestore()
 
+    /**
+     * Requests the [EngineSession] retrieve the current site's translation preference.
+     */
+    abstract fun getNeverTranslateSiteSetting(
```

Comment:
nit: Not sure if the docs or the name of the function is misleading, could you help me to clarify? 

Doc says get the current site's translation preference but the function indicates `getNeverTranslateSiteSetting`. Should the function be `getTranslationSetting`?

Reply:
Thanks, the docs are misleading, I updated to make it clear it is the `never` setting. (It'll eventually correspond to [this in the Figma for more info too](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?type=design&node-id=7202-99696&mode=design&t=VQubXcTivqPukHr6-0).)